packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -76,7 +76,7 @@ class TimelineScreenBodyState extends State<TimelineScreenBody> {
             axis: Axis.vertical,
             firstChild: TimelineFlameChart(),
             secondChild: EventDetails(),
-            initialFirstFraction: 0.8,
+            initialFirstFraction: 0.6,
           ),
         ),
```

Comment:
add a test validating this code. you already have the stub data to make that easy :)